pytest/pytest.go

```diff
@@ -12,8 +12,10 @@ import (
 	"testing"
 
 	_ "github.com/go-python/gpython/builtin"
+	_ "github.com/go-python/gpython/sys"
```

Comment:
This appears to duplicate the "sys" import?

Reply:
Done!